code/modules/mob/living/basic/jungle/venus_human_trap.dm

```diff
@@ -163,74 +161,82 @@
 	faction = list(FACTION_HOSTILE,FACTION_VINES,FACTION_PLANTS)
 	initial_language_holder = /datum/language_holder/venus
 	unique_name = TRUE
-	/// A list of all the plant's vines
-	var/list/vines = list()
-	/// The maximum amount of vines a plant can have at one time
-	var/max_vines = 4
-	/// How far away a plant can attach a vine to something
-	var/vine_grab_distance = 5
-	/// Whether or not this plant is ghost possessable
-	var/playable_plant = TRUE
+	speed = 1.2
+	melee_attack_cooldown = 1.2 SECONDS
+	ai_controller = /datum/ai_controller/basic_controller/human_trap
+	///how much damage we take out of weeds
+	var/no_weed_damage = 20
```

Comment:
tfw no weed 😔

Reply:
honestly adding a special interaction with cannabis would be funny